### PR TITLE
Refs [TASK][35927] Add splash screen, library, bottom nav

### PR DESCRIPTION
**Pull request by huydd-2678**
## Related Tickets
- [#Task 35927: Init package,base project apply MVP, splash screen](https://edu-redmine.sun-asterisk.vn/issues/35927)
## WHAT
- Add splash screen
- Add libraries
- Add icons
- Create bottom navigation
## Evidence (Screenshot or Video)
Capture

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -42,4 +53,31 @@ dependencies {
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+
+    implementation 'io.reactivex.rxjava3:rxjava:3.0.8'
+    implementation 'io.reactivex.rxjava3:rxandroid:3.0.0'
+    implementation 'io.reactivex.rxjava3:rxkotlin:3.0.1'
+    implementation 'com.jakewharton.rxbinding4:rxbinding:4.0.0'
```
Comment (thonc-2135) [inline app/build.gradle:60]: thêm lib ngoài lm j thế

File: app/src/main/res/navigation/navigation_graph.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/navigation_graph">
+
+
```
Comment (thonc-2135) [inline app/src/main/res/navigation/navigation_graph.xml]: remove blank

File: app/build.gradle (hunk 1 above)
Comment (huydd-2678) [reply app/build.gradle:60]: project 2 em dùng rx để làm ạ

File: app/src/main/res/navigation/navigation_graph.xml (hunk 2 above)
Comment (huydd-2678) [reply app/src/main/res/navigation/navigation_graph.xml]: Em ngồi xem đi xem lại cuối cùng lại sót đoạn này chưa remove

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**